msl-states.c

```diff
@@ -0,0 +1,167 @@
+#include <stdint.h>
+#include "msl-commands.h"
+#include "msl-states.h"
+
+// Initialitiation
+
+// to state E2+E3 if !AINI
+//          C8    if AINI
+
+uint8_t state_80_TI06_CU01(struct ge *ge) { return !ge->AINI; }
+uint8_t state_80_TI06_CU03(struct ge *ge) { return ge->AINI; }
+uint8_t state_80_TI06_CU05(struct ge *ge) { return !ge->AINI; }
+
+const struct msl_timing_chart state_80[] = {
+    { TO30, CI19, 0 },
+    { TO30, CO96, 0 },
+    { TO30, CO97, 0 },
+    { TO40, CO00, 0 },
+    { TO40, CO02, 0 },
+    { TO50, CI32, 0 },
+    { TI05, CI05, 0 },
+    { TI05, CI08, 0 },
+    { TI06, CI76, 0 },
+    { TI06, CI80, 0 },
+    { TI06, CI81, 0 },
+    { TI06, CI82, 0 },
+    { TI06, CU01, state_80_TI06_CU01 },
+    { TI06, CU03, state_80_TI06_CU03 },
+    { TI06, CU05, state_80_TI06_CU05 },
+    { TI06, CU06, 0 },
+    { MAX_CLOCK, 0 }
```

Comment:
shouldn't it be `{ MAX_CLOCK, 0, 0 }`?

however if we're OK with implicit inits, we can remove the trailing `, 0` in states too!